components/test_coprocessor/src/table.rs

```diff
@@ -74,14 +95,16 @@ impl Table {
         idx_info
     }
 
-    pub fn get_select_range(&self) -> KeyRange {
+    /// Create a `KeyRange` which select all records in current table.
+    pub fn record_select_all(&self) -> KeyRange {
```

Comment:
When I see this name, I expect it to return all the records in the table, instead of returning a `KeyRange` to select all records. So I suggest renaming to something like `get_record_range_all`.

Reply:
Good idea!